subprojects/core/src/main/java/org/gradle/internal/service/scopes/CoreBuildSessionServices.java

```diff
@@ -140,11 +138,6 @@ BuildSessionScopeFileTimeStampInspector createFileTimeStampInspector(BuildTreeSc
         return new BuildSessionScopeFileTimeStampInspector(workDir);
     }
 
-    @Provides
-    ScriptSourceHasher createScriptSourceHasher() {
```

Comment:
ℹ️  This has been moved to the `Global` scope, because it does not depend on anything